app/src/main/res/layout/activity_main.xml

```diff
@@ -1,47 +1,50 @@
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".MainActivity"
     android:background="#333333">
 
-    <ListView
-        android:layout_width="fill_parent"
-        android:layout_height="wrap_content"
-        android:id="@+id/listView"
-        android:layout_above="@+id/linearLayout" />
-
     <LinearLayout
         android:orientation="horizontal"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:id="@+id/linearLayout"
-        android:gravity="bottom|center"
-        android:layout_alignParentBottom="true"
-        android:layout_alignParentLeft="true"
-        android:layout_alignParentStart="true">
+        android:layout_alignParentBottom="true" >
 
         <ImageButton
-            android:layout_width="wrap_content"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:id="@+id/choresClaimed"
-            android:background="@mipmap/chores_claimed" />
+            android:background="@mipmap/chores_claimed"
+            android:layout_weight="1"/>
 
         <ImageButton
-            android:layout_width="wrap_content"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:id="@+id/choresUnclaimed"
-            android:background="@mipmap/chores_unclaimed" />
+            android:background="@mipmap/chores_unclaimed"
+            android:layout_weight="1"/>
 
         <ImageButton
-            android:layout_width="wrap_content"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:id="@+id/choresOverdue"
-            android:background="@mipmap/chores_overdue" />
+            android:background="@mipmap/chores_overdue"
+            android:layout_weight="1"/>
 
         <ImageButton
-            android:layout_width="wrap_content"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:id="@+id/choresEdit"
-            android:background="@mipmap/chores_overdue" />
+            android:background="@mipmap/chores_overdue"
+            android:layout_weight="1"/>
     </LinearLayout>
+
+    <ListView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
```

Comment:
also, wrap_content on ListView doesn't work. It makes the ListView disappear. ListView is always match_parent for height (and width)